README.md

```diff
@@ -1,7 +1,9 @@
-Orleans - Distributed Actor Model
+<p align="center">
+  <img src="https://github.com/dotnet/orleans/blob/gh-pages/assets/logo_full.png" alt="Orleans logo" width="600px"> 
+</p>
```

Comment:
Also resulting view can be observed by clicking this button: 

![chrome_2016-08-04_16-45-12](https://cloud.githubusercontent.com/assets/5787619/17404000/d7554926-5a62-11e6-92cb-04feb3ed8cd6.png)